index.html

```diff
@@ -1,43 +1,239 @@
+<body>
     <header>
-      <nav>
-        <a class="logo">My logo</a>
-        <div>
-          <i class="fa-solid fa-bars menu-bar"></i>
-        </div>
-      </nav>
+        <nav>
+            <a class="logo">My logo</a>
+            <div>
+                <i class="fa-solid fa-bars menu-bar"></i>
+            </div>
+        </nav>
     </header>
 
     <section class="introduction">
-      <h1 class="title"> I'm Vishal, Glad to See You!</h1>
-      <p class="description">I'm a software developer! I can help you build a product, feature or website. Look
-        through some of my work and experience. If you like what you see and have a project you need coded, don't
-        hestiate to contact me.</p>
-  
-      <h2 class="social order">LET'S CONNECT</h2>
+        <h1 class="title"> I'm Vishal <br>Glad to See <br>You!</h1>
+        <p class="description">I'm a software developer! I can help you build a product, feature or website. Look
+            through some of my work and experience. If you like what you see and have a project you need coded, don't
+            hestiate to contact me.</p>
+
+        <h2 class="social order">LET'S CONNECT</h2>
+
+        <ul class="social-links">
+            <li><a href="https://twitter.com/Vismiss30" target="_blank" rel="noopener"><i
+                        class="fa-brands fa-twitter li1"></i></a></li>
+            <li><a href="https://www.linkedin.com/in/vishal-kumar-mishra-0a5226232/" target="_blank" rel="noopener"><i
+                        class="fa-brands fa-linkedin li2"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-m li3"></i></a></li>
+            <li><a href="https://github.com/Vishal30M" target="_blank" rel="noopener"><i
+                        class="fa-brands fa-github li4"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-angellist li5"></i></a></li>
+        </ul>
+
+    </section>
+
+    <section class="work-section">
+        <div class="section1">
+          <div class="right-block">
+            <img class="img-section1" src="image/snap3.png" alt="project image">
+          </div>
+          <div class="left-block">
+            <div class="experience">
+              <h1 class="section-title">Tonic</h1>
+              <ul class="list">
+                <li class="list1">CANOPY</li>
+                <li class="list2">Back End Dev</li>
+                <li class="list3">2015</li>
+              </ul>
+            </div>
+            <p class="section-description0">A daily selection of privately personalized reads; no accounts or
+              sign-ups
+              required.</p>
+            <ul class="tags">
+              <li class="li11">
+                <article class="text1">html</article>
+              </li>
+              <li class="li12">
+                <article class="text2">css</article>
+              </li>
+              <li class="li13">
+                <article class="text3">javascript
+              </li>
+            </ul>
```

Comment:
- [ ] Please make sure that the design of this section matches the Figma design

### Your Design
![Screenshot from 2023-01-14 00-21-32](https://user-images.githubusercontent.com/92303891/212440913-8b05fb97-4c1d-441b-891f-833c1fe2a769.png)
### Figma Design
![Screenshot from 2023-01-14 00-26-26](https://user-images.githubusercontent.com/92303891/212441080-cf7ede16-8402-4d71-97b7-bd8d6c34672e.png)